app/config/errors.php

```diff
@@ -847,5 +847,20 @@
         'description' => 'Message with the requested ID has already been scheduled for delivery.',
         'code' => 400,
     ],
+    Exception::MESSAGE_TARGET_NOT_EMAIL => [
+        'name' => Exception::MESSAGE_TARGET_NOT_EMAIL,
+        'description' => 'Message with the target ID is not an Email target:',
+        'code' => 400,
+    ],
+    Exception::MESSAGE_TARGET_NOT_SMS => [
+        'name' => Exception::MESSAGE_TARGET_NOT_SMS,
+        'description' => 'Message with the target ID is not an SMS target:',
+        'code' => 400,
+    ],
+    Exception::MESSAGE_TARGET_NOT_PUSH => [
+        'name' => Exception::MESSAGE_TARGET_NOT_PUSH,
+        'description' => 'Message with the target ID is not a Push target:',
+        'code' => 400,
+    ],
```

Comment:
Let's use lower case for the target types (except SMS): email target, SMS target etc.

Reply:
updated this